### PR TITLE
Add entry to existing order

### PR DESCRIPTION
**Pull request by kprakobkit**

### CODE REVIEW

File: src/components/Entry.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ class Entry extends Component {
     return dom.tr(
       { className: 'order-entry' },
       dom.td({ className: 'entry-name' }, dom.h2(null, this.props.name)),
-      dom.td({ className: 'entry-comment' }, dom.h3(null, this.props.comment)),
+      dom.td({ className: 'entry-comment' }, dom.h2(null, dom.small(null, this.props.comment))),
```
Comment (achiu8) [inline src/components/Entry.js:22]: Why this way?

File: test/components/OrderDetails_spec.js [hunk 2]
```diff
@@ -52,5 +55,20 @@ describe('OrderDetails', () => {
     expect(entryStatuses[0].textContent).to.contain('OPEN');
     expect(entryStatuses[1].textContent).to.contain('DELIVERED');
   });
+
+  it('toggles add entry', () => {
+    const props = { orders: [{ status: 'open', id: 1, entries: [] }], params: { id: 1 }, changeEntryStatus, loadItems, addEntriesToOrder };
+    const component = renderIntoDocument(OrderDetails(props));
+    const toggleForm = findRenderedDOMComponentWithClass(component, 'toggle-add-entry');
+    let addMoreEntry = scryRenderedDOMComponentsWithClass(component, 'add-more-entry');
```
Comment (achiu8) [inline test/components/OrderDetails_spec.js]: Where's the `add-more-entry` class being used? What's this supposed to find? Should we be finding the same component as in line 69 to show it doesn't exist before clicking, but exists after?

File: server/action_creators.js [hunk 3]
```diff
@@ -70,6 +70,16 @@ export function changeEntryStatus(orderId, entryIndex, status) {
   };
 }
 
+export function addEntriesToOrder(orderId, newEntries) {
+  return (dispatch, getState) => {
+    return Order.addEntries(orderId, newEntries).then(() => {
+      return Order.getOrders();
```
Comment (achiu8) [inline server/action_creators.js:76]: This feels a little weird, but is prob fine..

File: src/components/MasterItems.js [hunk 4]
```diff
@@ -62,6 +75,39 @@ class MasterItems extends Component {
           'Add Item'
         )
       ),
+      dom.div(
```
Comment (achiu8) [inline src/components/MasterItems.js:78]: This is just moved from `NewOrder` component right? Seems like we just kinda moved everything over haha..

Comment (kprakobkit) [inline src/components/MasterItems.js:78]: ya...i think we can break down the master items into smaller components.

File: src/components/Entry.js (hunk 1 above)
Comment (kprakobkit) [inline src/components/Entry.js:22]: I tried adding the .small class but it doesn't style it the way I want.

http://getbootstrap.com/css/#type

**Pull request merged**